extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -85,11 +85,42 @@
   background-color: rgba(0,0,0, 1) !important;
 }
 
+.-amp-lightbox-viewer-mask {
+  z-index: 2147483643 !important;
+}
+
+.-amp-lightbox-viewer-gallery-mask {
+  display: none !important;
+  z-index: 2147483646 !important;
+}
+
+.-amp-lightbox-viewer[gallery-view] .-amp-lightbox-viewer-gallery-mask {
+  display: block !important;
+}
+
+.-amp-lightbox-viewer-gallery-grid {
+  margin-top: 50px;
+  height: 100vh;
+  width: 100vw;
+  display: flex 0 0 auto;
+}
+
+.-amp-lightbox-viewer-gallery-thumbnail {
+  display: inline-block;
+  width: 29vw;
+  height: 29vw;
+  margin: 2%;
+  overflow: hidden;
+  text-align: center;
+}
+
 /* Controls */
 .amp-lightbox-viewer-button-next,
 .amp-lightbox-viewer-button-prev,
-.amp-lightbox-viewer-button-close {
-  z-index: 2147483646 !important;
+.amp-lightbox-viewer-button-close,
+.amp-lightbox-viewer-button-gallery,
+.amp-lightbox-viewer-button-back{
+  z-index: 2147483645 !important;
```

Comment:
we should keep the button z-index above gallery, instead use the `.-amp-lightbox-viewer[gallery-view]` rules to hide/show controls